common/Unit.cpp

```diff
@@ -695,10 +698,6 @@ UnitKit::~UnitKit() {}
 
 UnitKit& UnitKit::get()
 {
-#if MOBILEAPP
-    if (!GlobalKit)
-        GlobalKit = new UnitKit("UnitKit");
-#endif
```

Comment:
Why don't you keep this inside an `if (Util::isMobileApp())` block?

Ah, I see, now calls to `get()` are changed to be `!Util::isMobileApp() && ...`, which means you don't need this.

Reply:
Actually it's because now isKitInProcess is always true in mobile, so next if is enough.